### PR TITLE
add instructions on how to run locally

### PR DESCRIPTION
**Pull request by jburel**
run the notebook locally using conda

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,29 +2,63 @@
 [![Actions Status](https://github.com/IDR/idr-notebooks/workflows/repo2docker/badge.svg)](https://github.com/ome/idr-notebooks/actions)
 # idr-notebooks
 
+A set of Python Notebooks to demonstrate how to access the images and metadata from the [Image Data Resource (IDR)](https://idr.openmicroscopy.org), including features and all descriptive tags.
+
+Full access to IDR metadata and images is provided through the standard OMERO API, documentation for which can be found [here](https://docs.openmicroscopy.org/latest/omero5.4/developers/index.html), with the Python bindings found specifically [here](https://docs.openmicroscopy.org/latest/omero5.4/developers/Python.html). The notebooks in this repository are meant to exemplify the use of that API in the context of the IDR, and the sort of queries that can be done. In particular, they show how to reproduce Figure 1 and Figure 2 of the paper.<sup>[1](#footnote1)</sup> They also make use of the [scipy](https://www.scipy.org/) ecosystem, including [pandas](https://pandas.pydata.org).
```
Comment (pwalczysko) [inline README.md:7]: ``...and the sort of queries that can be done. `` -> ``...and introduces the queries that this API enables.``
``... of the paper.<sup>[1](#footnote1)</sup>`` -> ``... of the original publication which introduced IDR <sup>[1](#footnote1)</sup>``

**Comment by pwalczysko** [comment]
Typo inside the notebooks table - add space before ``with``, see screenshot below
Screenshot 2022-06-22 at 17 04 43

File: README.md [hunk 2]
```diff
@@ -2,29 +2,63 @@
 [![Actions Status](https://github.com/IDR/idr-notebooks/workflows/repo2docker/badge.svg)](https://github.com/ome/idr-notebooks/actions)
 # idr-notebooks
 
+A set of Python Notebooks to demonstrate how to access the images and metadata from the [Image Data Resource (IDR)](https://idr.openmicroscopy.org), including features and all descriptive tags.
```
Comment (pwalczysko) [inline README.md:5]:
```suggestion
A set of Python Notebooks to demonstrate how to access the images and metadata from the [Image Data Resource (IDR)](https://idr.openmicroscopy.org), including the features and all of the descriptive tags.
```

Not sure what the descriptive tags are, maybe we can reformulate as above.

File: README.md [hunk 3]
```diff
@@ -2,29 +2,63 @@
...
 
-To build locally:
 
- * Install [Docker](https://www.docker.com/) if required
- * Create a virtual environment and install repo2docker from PyPI.
- * Clone this repository
- * Run ``repo2docker``. 
- * Depending on the permissions, you might have to run the command as an admin
+An additional benefit of installing the requirements locally is that you
+can then use the tools without needing to launch Jupyter itself.
```
[the remark this answers is not in the document]
Comment (jburel) [reply README.md:60]: there is no ``scripts`` folder in this repo

**Comment by jburel** [comment]
The aim of this PR was not a full review of the readme. It only focused on the installation section
We should focus on that for this PR, we can review it in a follow-up PR

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (pwalczysko) [reply README.md:60]: How would you then use the tools ?

**Comment by pwalczysko** [comment]
> We should focus on that for this PR, we can review it in a follow-up PR

OK. Then maybe just act only on the comments made on parts you added please ?

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jburel) [reply README.md:60]: copy/paste so I will remove that

**Comment by jburel** [comment]
done

**Review by pwalczysko — APPROVED** [review]

**Pull request merged**